src/redux/reducers/genes/index.js

```diff
@@ -54,6 +58,12 @@ const genesReducer = (state = getInitialState(), action) => {
     case MARKER_GENES_LOADING: {
       return markerGenesLoading(state, action);
     }
+    case DOWNSAMPLED_GENES_LOADING: {
+      return downsampledGenesLoading(state, action);
+    }
+    case DOWNSAMPLED_GENES_LOADED: {
+      return downsampledGenesLoaded(state, action);
```

Comment:
To manage the new `.downsampled` part of redux.

I also considered using the same reducers and checking with a `downsampled` bool in the payload of the action, but since the reducers work relatively differently, it didn't seem to be the most clear solution compared to this full explicit separation.